internal/server/middlewares.go

```diff
@@ -106,13 +109,31 @@ func requestLogger(lg *zap.Logger) middleware {
 			t := time.Now()
 			span := trace.FromContext(req.Context())
 
+			buf, err := io.ReadAll(req.Body)
+			if err != nil {
+				lg.Error("error reading request body: %v", zap.String("error", err.Error()))
```

Comment:
let's make this only for `debug` or `trace` level?

Reply:
this is done